### DISCUSSION TITLE
Settings Page

### DISCUSSION THREAD

**Discussion by 4irm** [Blackymas/NSPanel_HA_Blueprint#1916] [Q&A] [answered]
Dear community,

I did some research, however, without success. Question: How to access the settings page (brightness, reboot, etc.)?

I do remember seeing a little gear on the home page in one of the documentation pages, but I don't have that one. I am on 4.3.0 on a white EU Panel.

Thanks for your support!

**Comment by plasterer41** [comment, marked as answer]
Hold your finger on the hour numbers on the clock homepage. 👍🏼
